application/src/main/java/net/corda/v5/application/messaging/FlowSessionConfiguration.java

```diff
@@ -0,0 +1,62 @@
+package net.corda.v5.application.messaging;
+
+import java.time.Duration;
+
+/**
+ * Flow session configuration. Instances should be created using {@link Builder}.
+ */
+public class FlowSessionConfiguration {
```

Comment:
Should this and the builder be `final`?

Reply:
This class will be container for session configuration properties, so when we'll need another property (similar to `requireClose` or `timeout`) in the future we'll add fields to this class. Since we don't plan to extend it, I'll change classes to be final.